backend/src/graphql/model/User.ts

```diff
@@ -37,6 +41,8 @@ export class User {
       this.gmsPublishName = user.gmsPublishName
       this.humhubPublishName = user.humhubPublishName
       this.gmsPublishLocation = user.gmsPublishLocation
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
```suggestion
```
not needed any longer, because your argument is safe, isn't it? When I remove it, I don't get linting errors.